src/test/java/integration/SearchApiTest.java

```diff
@@ -106,4 +106,39 @@ void closeSearchOperations_Success() {
 
         assertTrue(closeSearchResponse.value);
     }
+
+     @Test
+    void getSearchResults_SimplePaging() throws InterruptedException {
+        int maxPageSize = 1;
+
+        AdvancedSearchRequest request = new AdvancedSearchRequest();
+        request.searchCommand = "({LF:Basic ~= \"*\", option=\"DFANLT\"})";
+
+        AcceptedOperation searchOperationResponse = client.createSearchOperation(repoId, request).join();
+
+        String searchToken = searchOperationResponse.token;
+        assertTrue(searchToken != null && !searchToken.trim().isEmpty());
+
+        TimeUnit.SECONDS.sleep(5);
+
+        CompletableFuture<ODataValueContextOfIListOfEntry> searchResultResponse = client.getSearchResults(repoId, searchToken, null, null, null, null, String.format("maxpagesize=%d", maxPageSize), null, null, null, null, null, null);
+        ODataValueContextOfIListOfEntry searchResults = searchResultResponse.join();
+
+        assertNotNull(searchResults);
+        if(searchResults.value.isEmpty()) {
+            return; // There's no point testing if we don't have any such item.
+        }
+
+        String nextLink = searchResults._atOdataNextLink;
+        assertNotNull(nextLink);
+        assertTrue(searchResults.value.size() <= maxPageSize);
+
+        // Paging request
+        searchResultResponse = client.getSearchResultsNextLink(nextLink, maxPageSize);
```

Comment:
when i checkout this branch and pull the latest version, i get a build error saying that this function does not exist, is this expected?

![image](https://user-images.githubusercontent.com/103658932/191362151-5a7142cc-dab9-4c5c-bb92-a441abde0a3c.png)

Reply:
The code in Program.java has two lines,
    Processor.transformModelClass("\\..\\script\\client\\model");
    Generator.generateApiClasses("\\..\\script\\output.json");
Which I think the second one is commented. In order to generate the code, comment the first one, and uncomment the second one. Then run Program.java.